web/src/lib/client/manager.js

```diff
@@ -46,4 +51,27 @@ export default class ManagerClient extends Client {
     const proxy = await this.proxy(MANAGER_IFACE);
     return proxy.Status;
   }
+
+  /**
+   * Register a callback to run when properties in the Actions object change
```

Comment:
I think it is not about actions....and I would probably split it to two methods.